build-system/shadow-viewer.js

```diff
@@ -144,7 +144,7 @@ const SCRIPT = `
 };
 `;
 
-const renderShadowViewer = ({src, baseHref, port = 8000}) =>
+const renderShadowViewer = ({src, baseHref}) =>
```

Comment:
@alanorozco Can you review this change since you originally added this code?

Reply:
Cherry-picked to separate PR: #24066.